src/test/java/uk/gov/companieshouse/documentsigningapi/controller/SignDocumentControllerIntegrationTest.java

```diff
@@ -214,6 +226,9 @@ void signPdfWithUnknownDocumentLocation() throws Exception {
         final var signPdfRequestDTO = createSignPdfRequest(unsignedDocumentLocation);
 
         final var resultActions = mockMvc.perform(post("/document-signing/sign-pdf")
+                        .header(ERIC_IDENTITY_HEADER_NAME, ERIC_IDENTITY_HEADER_VALUE)
+                        .header(ERIC_IDENTITY_TYPE_HEADER_NAME, ERIC_IDENTITY_HEADER_VALUE)
+                        .header(ERIC_AUTHORIZED_KEY_ROLES, ERIC_AUTHORIZED_KEY_ROLES_VALUE)
```

Comment:
The indentation on these lines appears to differ from lines 172-174, can it be consistent throughout please?